src/ui-todo.c

```diff
@@ -314,7 +314,28 @@ void ui_todo_popup_item(void)
 	if (!item)
 		return;
 
-	item_in_popup(NULL, NULL, item->mesg, _("TODO:"));
+	if (item->note) {
+		/* Assign a sane default note size that will cleanly
+		 * truncate long notes */
+		size_t note_size = 3500;
+		char note[note_size];
+
+		char *notepath;
+		FILE *fp;
+		asprintf(&notepath, "%s%s", path_notes, item->note);
+
+		fp = fopen(notepath, "r");
+		note_read_contents(note, note_size, fp);
+		fclose(fp);
+
+		char *msg;
+
+		asprintf(&msg, "%s\n\nNOTES:\n%s", item->mesg, note);
+
+		item_in_popup(NULL, NULL, msg, _("TODO:"));
+	}
+	else
```

Comment:
Style: Please put else on the same line as the closing braces; use braces for an else-block if the if-block has braces (even if the body consists of just a single line). This line should be "} else {".